react/components/ChannelContents/index.js

```diff
@@ -54,16 +52,14 @@ export default class ChannelContents extends PureComponent {
                 onAddBlock={this.handleAddBlock}
               />
             </GridItem>
-
-            {newBlocks.length > 0 &&
-              <ChannelContentsSet
-                id={id}
-                skeleton={newBlocks}
-              />
-            }
           </React.Fragment>
         }
 
+        <ChannelContentsAppended
+          id={id}
+          pendingSkeleton={pendingSkeleton}
+        />
```

Comment:
This winds up making way more sense: we don't want to rechunk the skeleton and we'd get the exact same result if we did.

Reply:
Nice, yes.